docs/website/blog/2022-05-02-odo-v3-alpha1.md

```diff
@@ -0,0 +1,327 @@
+---
+title: odo v3-alpha1 Released
+authors:
+- name: Parthvi Vala
+  url: https://github.com/valaparthvi
+  image_url: https://github.com/valaparthvi.png
+- name: Philippe Martin
+  url: https://github.com/feloy
+  image_url: https://github.com/feloy.png
+tags: ["release"]
+slug: odo-v3-alpha1-release
+---
+
+`v3-alpha1` of odo has been released!
+
+<!--truncate-->
+
+`odo` is a tool that aims to simplify the life of developers working on cloud-native applications.
+
+Thanks to the emergence of the [Devfile](https://devfile.io/) open standard, which has been accepted as a CNCF Sandbox project (January 2022), odo v3 is now entirely based on this open standard.
```

Comment:
odo v3 is now entirely based on this open standard.

Reply:
I have added `entirely`, because v2 was already based on devfile, but also on s2i.